### PR TITLE
Te addition pr

### PR DESCRIPTION
**Pull request by suchmememanyskill**
This needs a review.

### CODE REVIEW

File: docs/user_guide/emummc/partitioning_sd.md [hunk 1]
```diff
@@ -3,77 +3,48 @@
...
+7. Navigate to `Exit` and press the power button to enter the Exit menu
+8. Navigate to `Reboot to RCM` and press the power button to reboot to RCM. It's now safe to eject your sd card for the next part of the guide
 
 &nbsp;
 
 #### [Continue to SD Preparations <i class="fa fa-arrow-circle-right fa-lg"></i>](sd_preparation.md)
 
 -----
-## Linux Instructions
+## Manual Linux Preperations
```
Comment (noirscape) [inline docs/user_guide/emummc/partitioning_sd.md]: Just remove the linux instructions. If TegraExplorer can take care of this, it's not needed.

File: docs/user_guide/emummc/making_emummc.md [hunk 2]
```diff
@@ -9,6 +9,7 @@
 
 !!!tip ""
     1. Enter RCM and inject the Hekate payload
+        - If you forgot how to do this, re-read the [sending payload](sending_payload.md) section of the guide.
```
Comment (noirscape) [inline docs/user_guide/emummc/making_emummc.md]: Feels redundant. (Includes repetitions later down this PR.)

I don't really see the need for this, we should be able to assume that users who follow our guide linearly maintain the knowledge from previous pages.

File: docs/user_guide/emummc/partitioning_sd.md (hunk 1 above)
Comment (suchmememanyskill) [reply docs/user_guide/emummc/partitioning_sd.md]: Linux instructions are now on a completely separate page

File: docs/user_guide/emummc/making_emummc.md (hunk 2 above)
Comment (suchmememanyskill) [reply docs/user_guide/emummc/making_emummc.md]: All the links back to sending_payload.md have been removed from making_emummc.md

**Pull request merged**